test/integration/targets/vmware_guest_serial_port/tasks/main.yml

```diff
@@ -0,0 +1,100 @@
+# Test code for the vmware_guest_serial_port Operations.
+# Copyright: (c) 2019, Anusha Hegde <anushah@vmware.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- import_role:
+    name: prepare_vmware_tests
```

Comment:
You need to add:

```
  vars:
    setup_attach_host: true
    setup_datastore: true
```

- setup_attach_host: the ESXi will be attached to the vCenter
- setup_datastore: datastore are mounted on the ESXi